### PR TITLE
Begin v0->v1 migration with object builders

### PR DESCRIPTION
**Pull request by i509VCB**
This migrates object builders to v1, only two things need some consideration:

Old Registry events and whether we want to so some additional handling

v0 FabricMaterialBuilder needs some additional consideration since it can't be easily delegated to the v1 classes like many of the other objects.

This is relevant to #305, #493 and #528.

### CODE REVIEW

File: fabric-object-builders-v0/build.gradle [hunk 1]
```diff
@@ -1,7 +1,8 @@
 archivesBaseName = "fabric-object-builders"
-version = getSubprojectVersion(project, "0.1.3")
+version = getSubprojectVersion(project, "0.1.4")
 
 dependencies {
     compile project(path: ':fabric-api-base', configuration: 'dev')
     compile project(path: ':fabric-mining-levels-v0', configuration: 'dev')
+	compile project(path: ':fabric-object-builders-v1', configuration: 'dev')
```
Comment (Juuxel) [inline fabric-object-builders-v0/build.gradle:7]: Could you change the two lines above to use tabs as well?

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: fabric-object-builders-v0/build.gradle (hunk 1 above)
Comment (i509VCB) [reply fabric-object-builders-v0/build.gradle:7]: Yea that's an obvious one, lemme get a feed and I'll fix it

**Review by Juuxel — APPROVED** [review]

**Comment by i509VCB** [comment]
Targeting 1.16 now

**Comment by i509VCB** [comment]
Going to open a new PR due the shear destruction of the commit tree

**Comment by i509VCB** [comment]
Superseded by #537

**Pull request closed without merging**